pkg/cri/server/metrics.go

```diff
@@ -54,5 +59,9 @@ func init() {
 	containerStopTimer = ns.NewLabeledTimer("container_stop", "time to stop a container", "runtime")
 	containerStartTimer = ns.NewLabeledTimer("container_start", "time to start a container", "runtime")
 
+	imagePullingError = ns.NewLabeledCounter("image_pulling_error", "error count of image pulling by image name", "image_name")
```

Comment:
Using image name will make the cardinality too high. Prometheus' doc recommends to keep the cardinality lower than 10, so I guess registry name should be a better option here.

[1] https://prometheus.io/docs/practices/instrumentation/#do-not-overuse-labels

Reply:
```
	imagePulls = ns.NewLabeledCounter("image_pulling", "count of image pulling by registry", "status", "registry")
```
I changed to this.

Reply:
I don't think it's reasonable to assume that containerd will only pull from 10 distinct domains during its lifetime (which appears to be the actual distinction, not registries?).  Some container registry providers use many distinct domains (for example, Amazon ECR has a different domain name for each customer account).